src/main/scala/nl/knaw/dans/easy/stage/package.scala

```diff
@@ -40,4 +40,29 @@ package object stage {
       }
     }
   }.isSuccess
+
+
+  /*
+   Logic opperators that work with Try[Boolean]
+   */
+  implicit class TryLogic(val t: Try[Boolean]) extends AnyVal {
```

Comment:
but now we do:

* http://stackoverflow.com/a/4982668/2389405
* http://stackoverflow.com/questions/3307427/scala-double-definition-2-methods-have-the-same-type-erasure
* https://medium.com/byte-code/overcoming-type-erasure-in-scala-8f2422070d20#.gsbhi3f9x

if we eventually want to make this part of the dans-scala-lib, I guess we have to provide a full implementation here already.